wporg-gp-translation-events.php

```diff
@@ -200,4 +187,14 @@ function register_translation_event_js() {
 add_action( 'init', 'register_event_post_type' );
 add_action( 'add_meta_boxes', 'event_meta_boxes' );
 add_action( 'save_post', 'save_event_meta_boxes' );
-register_routes();
+
+if ( class_exists( GP::class ) ) {
```

Comment:
This should rather be done in a filter since it's not guaranteed that the GlotPress plugin will be loaded before this one.
```suggestion
add_action( 'gp_init', function() {
```

Reply:
That makes a lot of sense. I fixed it in 9e535543602fe00211ca8471f221e088a3daaf8a.